### ISSUE TITLE
protected *.key should stay protected and ask for PW every use

### ISSUE THREAD

**Issue by Prash3r** [SoftEtherVPN/SoftEtherVPN#963]
Hallo everyone,

individual certs Auth:
password protected certificate keys should ask for password on every use!

alternatively there should be an option to password encrypt/protect the config on disk to be asked to decrypt on every start/use. *.key in cleartext is a security issue

**Expected behavior:**
If i provide a password protected .key i want to be asked for the password every time it is used.
Maybe store password in RAM while process is running. 

Anyone exporting VPN Connection Settings should NOT be able to just use the config including the key if the key was originally provided with password protection.

I get why there would be an option to store the provided key pass, people are lazy, but the default should be "ask me every time". And if the .key is protected i would argue to not even make "remember" an option.

**Actual behavior:**
It strips the key of the password protection when creating the config and stores the cleartext key on hard drive AND in exportable *.vpn config files.Thus throwing the last bit of password check security out of the window. everyone with access to the config has instant access to the vpn.

**Comment by dnobori** [comment]
I agree with you very much. To implement it, a developer will need to write a new UI and protocol between the foreground GUI process and the background service. Personally I have no time to implement it immediately. I hope either you or someone will implement it since SoftEther VPN is open source and welcome to any contribution.


Until the feature will be implemented, you can protect the VPN Client Manager by the one-stop password, as the following screenshot. This protection mechanism is already implemented. The password will be asked every time start the VPN Client Manager.


![clip_1](https://user-images.githubusercontent.com/3041130/63138005-9d734680-c013-11e9-9bc0-766006ac81d3.jpg)
